README.md

```diff
@@ -14,16 +14,12 @@
   - [Badge](#badge)
 - [Definitions](#definitions)
 - [API](#api)
-  - [IPLD format utils](#ipld-format-utils)
-    - [`util.serialize(dagNode, callback)`](#utilserializedagnode-callback)
-    - [`util.deserialize(binaryBlob, callback)`](#utildeserializebinaryblob-callback)
-    - [`util.cid(binaryBlob[, options], callback)`](#utilcidbinaryblob-options-callback)
-  - [Local resolver methods](#local-resolver-methods)
-    - [`resolver.resolve(binaryBlob, path, callback)`](#resolverresolvebinaryblob-path-callback)
-    - [`resolver.tree(binaryBlob, callback)`](#resolvertreebinaryblob-callback)
```

Comment:
Large naming changes might be out of the scope here, but if it isn't  I would favor following terminology: